packages/shared-ui/src/components/canvas/components/node/node-component/status/NodeStatus.tsx

```
@@ -21,7 +21,7 @@ import React, { ReactElement } from 'react';
 
 import { ITaskStatus, ITaskState } from '../../../../types';
 import { PipelineActions } from '../../../../../../components/pipeline-actions';
-import { commonStyles } from '../../../../../../themes/styles';
+import { commonStyles } from 'shell/src/themes/styles';
```

ESLint

[error] 'shell/src/themes/styles' import is restricted from being used by a pattern. Only shell package sources are deep-importable in-tree (shell/src/<group>).

(no-restricted-imports)